### DISCUSSION TITLE
Hover affecting completely different element

### DISCUSSION THREAD

**Discussion by Psycarlo** [tailwindlabs/tailwindcss#9079] [Help]
**Hello**,

I am developing a Vue application with **TailwindCSS**  and **HeadlessUI**

I have a component that uses a HeadlessUI Menu component and I animate the Menu Button on hover with group and group-hover
On a completely different component I have an input that has a "Copy" button inside it so users can copy the text displayed.

**The problem:**
When I hover the first component, the color on the "Copy" text button changes. Components are not linked.
When I remove the group-hover:rotate... the problem disappears
What can I do?

**Comment by wongjn** [comment]
Could we get the code that reproduces your issue please? We can then look at troubleshooting from there!
